src/main/java/com/university/university/model/Course.java

```diff
@@ -0,0 +1,29 @@
+package com.university.university.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.bson.types.ObjectId;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.annotation.PersistenceConstructor;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.repository.Query;
+import org.springframework.data.repository.RepositoryDefinition;
+import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Indexed;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
```

Comment:
Please do not use Data annotation it import a lot of classes that we do not use, 
Only use the necessary annotations